source/reference/minio-server/minio-server.rst

```diff
@@ -2677,7 +2487,7 @@ variables.
       :start-after: start-minio-ad-ldap-server-addr
       :end-before: end-minio-ad-ldap-server-addr
 
-   This environment variable corresponds with the :mc-cmd:`mc idp ldap add server_addr` command.
+   This environment variable corresponds with :mc-cmd:`mc idp ldap add` with the :mc-cmd:`~mc idp ldap add server_addr` parameter.
```

Comment:
This wording is a variation on Daryl's original suggestion, to somewhat match the style of the `mc admin config` ones. Suggestions welcome.

Technically these LDAP envvars correspond with both `mc idp ldap add` and `mc idp ldap update`. But adding that makes the sentence a bit cumbersome. Worth doing anyway?

Reply:
```suggestion
   This environment variable corresponds with the :mc-cmd:`mc idp ldap add server_addr` parameter.
```

Reply:
I agree with Ravind to not worry about the update.
If it's set at an envvar, that overrides any configuration setting anyway.

Reply:
✅  search/replace fail
_sigh_